servicex/configuration.py

```diff
@@ -46,6 +47,35 @@ class Configuration(BaseModel):
     cache_path: Optional[str] = Field(alias="cache-path", default=None)
     shortened_downloaded_filename: Optional[bool] = False
 
+    @validator("cache_path", always=True)
+    def expand_cache_path(cls, v):
+        """
+        Expand the cache path to a full path, and create it if it doesn't exist.
+        Expand ${USER} to be the user name on the system. Works for windows, too.
+        :param v:
+        :return:
+        """
+        if not v:
+            v = "/tmp"
```

Comment:
Two questions about this line:
* does this work on windows?
* /tmp is machine wide, and you probably need something that is per-user. So you should do /tmp/<user> I think. At least, I have that code in the original and I had to do that for a reason... I think it had to do with wanting to clear the cache, and only clearing my stuff and not others that were on the same machine.